package.json

```diff
@@ -14,7 +14,6 @@
     "prettier-check": "prettier . --check",
     "prettier-fix": "prettier . --write",
     "tsc": "tsc && tsc --project tsconfig.no-implicit-any.json",
-    "tsc-watch": "tsc --watch",
```

Comment:
How can we run `tsc` in watch mode using `turbo`? :thinking:

Reply:
`turbo watch tsc`
It'll cache a bit more, and isn't any more characters.

Reply:
I'm getting this error when trying it:
```
• Packages in scope: 
• Running tsc in 0 packages
• Remote caching disabled
  × failed to connect to daemon
  ╰─▶ server is unavailable: channel closed
```

Reply:
Hmm. Ok, I'll revert it.